File: node/src/main/java/dev.httpmarco.polocloud.node/util/RandomUtils.java
package dev.httpmarco.polocloud.node.util;

import lombok.experimental.UtilityClass;

import java.util.Random;

@UtilityClass
public class RandomUtils {

    private final Random RANDOM = new Random();

    public int getRandomNumber(int max) {
        return RANDOM.nextInt(max);
    }
}

File: modules/rest/src/main/kotlin/dev/httpmarco/polocloud/modules/rest/utils/RandomUtils.kt
package dev.httpmarco.polocloud.modules.rest.utils

import java.security.SecureRandom

fun generateRandom(length: Int = 170): String {
    val chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"
    val random = SecureRandom()
    return (1..length)
        .map { chars[random.nextInt(chars.length)] }
        .joinToString("")
}